.github/workflows/main.yml

```diff
@@ -86,11 +86,15 @@ jobs:
           - '14.1' # Swift 5.7.1
     steps:
       - uses: actions/checkout@v2
+      - uses: apple-actions/import-codesign-certs@v2
+        with:
+          p12-file-base64: ${{ secrets.SIGNING_CERTIFICATE_BASE_64 }}
```

Comment:
nit: `_BASE64` would be a little more natural spelling, but no strong feelings on that.